src/main/kotlin/com/jetbrains/snakecharm/inspections/quickfix/AddIgnoredElementQuickFix.kt

```diff
@@ -1,52 +1,30 @@
 package com.jetbrains.snakecharm.inspections.quickfix
 
-import com.intellij.codeInsight.template.TemplateBuilderImpl
-import com.intellij.codeInspection.LocalQuickFixAndIntentionActionOnPsiElement
-import com.intellij.codeInspection.ex.InspectionProfileModifiableModel
-import com.intellij.openapi.editor.Editor
+import com.intellij.codeInsight.intention.LowPriorityAction
+import com.intellij.codeInspection.LocalQuickFix
+import com.intellij.codeInspection.ProblemDescriptor
 import com.intellij.openapi.project.Project
-import com.intellij.profile.codeInspection.InspectionProfileManager
 import com.intellij.psi.PsiElement
-import com.intellij.psi.PsiFile
 import com.jetbrains.snakecharm.SnakemakeBundle
 import com.jetbrains.snakecharm.inspections.SmkUnrecognizedSectionInspection
 
-class AddIgnoredElementQuickFix(element: PsiElement) : LocalQuickFixAndIntentionActionOnPsiElement(element) {
-    private val elementName = element.text
+class AddIgnoredElementQuickFix(section: PsiElement) : LocalQuickFix, LowPriorityAction {
+    private val sectionName = section.text
```

Comment:
`section: PsiElement` used only once for private field initialization, let's simply constructor to `class AddIgnoredElementQuickFix(private val sectionName: String)`